routing/routes.go

```diff
@@ -27,6 +27,24 @@ func NewRoutes() (r *Routes) {
 			"/api",
 			handlers.Root,
 		},
+		Route{
+			"FailExample",
+			"GET",
+			"/api/500",
```

Comment:
Is this something standard? I have never seen a REST api which has an http error code as part of its url.
Not sure what is the purpose of adding an example of failure to source code this way

Reply:
No, it isn't an standard. This is just an endpoint to show how the errors would respond to.
This endpoint, should be removed in a posterior iterations. That is illustrative.

Reply:
I don't think we need a temporary API to show errors. 
REST api sometimes returns errors of all kinds, it's standard, not sure what is this demonstrating here.
This added code is not something we should use and you say yourself it will be removed later.
So unless I am missing something - this is redundant